boards/arm/thingy91_nrf9160/CMakeLists.txt

```diff
@@ -12,4 +12,8 @@ endif()
 if(CONFIG_BOARD_THINGY91_NRF9160NS)
 	zephyr_library()
 	zephyr_library_sources(board_nonsecure.c)
+
+	# Use static partition layout to ensure consistency between builds.
+	# This is to ensure settings storage will be at the same location after the DFU.
+	set(PM_STATIC_YML_FILE pm_static.yml)
```

Comment:
Isn't the location of the storage partition fixed already? 🤔 
```
nvs_storage:
  placement: {before: [end]}
  size: CONFIG_PM_PARTITION_SIZE_NVS_STORAGE
```

Reply:
@lemrey  I am not sure entirely sure what you mean here. But this change is done to ensure backwards compatibility with an older version mcuboot that comes factory-programmed with thingy91.

See step 3 in https://devzone.nordicsemi.com/nordic/cellular-iot-guides/b/getting-started-cellular/posts/how-to-program-any-project-to-the-thingy-91-with-dfu

Reply:
@jtguggedal I am not sure how this is supposed to work. How do we reserve memory for the Modem library in Thingy when using a static PM configuration?

Reply:
Whatever is not reserved by a static configuration is still available for regular "dynamic" resolution.

Reply:
So this file is appended with `pm.yml.libmodem` and whatever else is enabled as long as there is space for it?

Reply:
Its not appended, but the areas reserved in the static configuration is removed from the "free area" before the dynamic resolving of the areas defined in `pm.yml.libmodem` starts. So your statement is sort of correct functionally